pycbc/workflow/coincidence.py

```diff
@@ -141,7 +141,11 @@ def create_node(self, coinc_files, tags=None):
 
         node = Node(self)
         node.set_memory(5000)
-        node.add_input_list_opt('--coinc-files', coinc_files)
```

Comment:
Let's not add this confusion. Can we not just send a list of one file, if that's the mode its operating in? (Or if we no longer need a list, make this a single file)

Reply:
I'm not sure about this, I'll give it a think...

Reply:
@spxiwh If possible, I would like to keep this change in.